packages/react-api/__tests__/hooks/gpu-filter-utils.test.js

```diff
@@ -0,0 +1,92 @@
+import { getGpuFilter, MAX_GPU_FILTERS } from '../../src/hooks/gpu-filter-utils';
+
+describe('gpu-filter-utils', () => {
+  test('correct GPU filter size', () => {
+    expect(MAX_GPU_FILTERS).toEqual(4);
+  });
+
+  test('correct filter values without time filter', () => {
+    const filters = {
+      storetype: {
+        in: {
+          values: ['Supermarket'],
+          owner: 'revenueByStoreType'
+        }
+      },
+      revenue: {
+        closed_open: {
+          values: [[1400000, 1500000]],
+          owner: 'storesByRevenue'
+        }
+      }
+    };
+    const featurePassesFilter = {
+      properties: {
+        storetype: 'Supermarket',
+        revenue: 1400001
+      }
+    };
+    const featureNotFilter = {
+      properties: {
+        storetype: 'Supermarket',
+        revenue: 100
+      }
+    };
+    const { filterRange, filterValueUpdateTriggers, getFilterValue } = getGpuFilter(
+      filters
+    );
+
+    expect(filterRange.length).toEqual(MAX_GPU_FILTERS);
+    filterRange.forEach((range, index) => {
+      const valueExpected = index === 0 ? [1, 1] : [0, 0];
```

Comment:
Why do we need positions 2 and 3 to have also [0, 0]. I find it a bit strange, from the test point of view.

I would probably stress in the test that the only relevant positions are positions 0 (common filters) and 1 (temporal filter). If you don't want to rename variables (eg for `const commonFilter = [1, 1]`, `const temporalFilter = [0, 0]`), then we should add at least a comment

Reply:
Because it's very important that positions 2 and 3 are [0, 0] because `MAX_GPU_FILTERS` is 4